### PR TITLE
Moddifed

### PR DESCRIPTION
**Pull request by noah4546**

### CODE REVIEW

File: src/bot.js [hunk 1]
```diff
@@ -11,32 +11,7 @@ const client = new Client({
 })
 
 async function run() {
-    db(client)
```
Comment (noah4546) [inline src/bot.js:14]: This isnt right

**Comment by daniel000111** [comment]
you suck

File: src/bot.js (hunk 1 above)
Comment (daniel000111) [reply src/bot.js:14]: you suck!

Comment (noah4546) [reply src/bot.js:14]: I fixed it

**Pull request closed without merging**